src/Parser/HavocDemonHunter/Modules/Features/CastEfficiency.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import CoreCastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+
+class CastEfficiency extends CoreCastEfficiency {
+  static CPM_ABILITIES = [
```

Comment:
It would be nice if *all* abilities were here in one form or another. As an example look at the Disc Priest CastEfficiency which has *rotational spell*, *cooldowns*, *spells* and *utility* categories. It even has Shackle Undead so that the list is complete.

Reply:
I'll add all abilities in a future PR.